README.md

```diff
@@ -27,6 +27,36 @@ Installation instructions for the CLI
 
 This repository does not need any specific configuration.
 
+
+### Docker
+
+To build a local image for Zonemaster CLI you need a [local Zonemaster Engine
+base image].
+
+Build a new local base image:
+
+```sh
+make all dist docker-build
+```
+
+Tag the local base image with the current version number:
+
+```sh
+make docker-tag-version
+```
+
+Tag the local base image as the latest version:
+
+```sh
+make docker-tag-latest
+```
+
+Test a zone using the local base image:
+
+```sh
+docker run --rm zonemtaster/cli:local zonemaster.net
```

Comment:
We could add `-t` if we want the spinner back.

Reply:
I will include that in examples.